### DISCUSSION TITLE
Adding option to change view of "Show Hidden Item" button on Taskbar

### DISCUSSION THREAD

**Discussion by vippium** [valinet/ExplorerPatcher#1430] [Ideas]
It will be great if Dev's add a option to select the type of view for "Show hidden icons" button on Win10 Taskbar. Like we can switch between flyouts to Win11,Win10 and win7. It will be great if dev's add this because I like the new menu of that button but Unfortunately the only way is to Win11 Taskbar to that. (which I personally don't like). Here are ss of these:

**Flyout on Win11 Taskbar ;**
![2023-05-09_18h41_52](https://github.com/valinet/ExplorerPatcher/assets/93650504/c7ce0cdb-d84e-43cd-af71-2a06fba488d5)

**Flyout on Win10 Taskbar:**
![2023-05-09_18h43_07](https://github.com/valinet/ExplorerPatcher/assets/93650504/6c99cb6c-8788-4737-b1a7-e9cd3078dfb6)

**After Adding option:**
![2023-05-09_18h56_09](https://github.com/valinet/ExplorerPatcher/assets/93650504/9b5d8872-c8b7-47aa-8e65-80ed6a5322a2)

**Comment by pyrates999** [comment]
I just have it set to show all icons in the system tray in taskbar settings.

**Reply by vippium** [reply]
I don't set this bcoz it uses more space if I have more apps on taskbar

**Reply by CorruptionVR** [reply]
hey, how can I enable the "show hidden icons" menu with ExplorerPatcher? because I can't seem to enable it. all of the icons are showing on the taskbar, and it takes up a lot of space.

edit: nevermind, I just had a setting enabled.
